packages/server/src/share/share.type.ts

```diff
@@ -0,0 +1,20 @@
+import { InputType, Field } from '@nestjs/graphql';
```

Comment:
This should be converted to an enum of:
- encrypted
- number

And then apply that type to Share Object:
https://github.com/multiparty/MassTech/pull/7/files#diff-322bdbf79e510a4cb3dee10ac1445d35499fd4cd606dcd34ec1dd7585d41b622R13